docs/polaris/compiling-and-linking/polaris-programming-models.md

```diff
@@ -28,7 +28,7 @@ A summary of available GPU programming models and relevant compiler flags is sho
 | HIP* | -- | -- | -- | -- |
 | OpenACC | -- | -acc | -- | -- |
 | OpenCL* | -- | -- | -- | -- |
-| OpenMP | --| -mp=gpu | -fopenmp-targets=nvptx64 | -- |
+| OpenMP | --| -mp=gpu | -fopenmp-targets=nvptx64 --offload-arch=sm_80 -fopenmp-offload-mandatory | -- |
```

Comment:
`-fopenmp-targets=nvptx64` no longer needed. It can be derived from `--offload-arch`

Reply:
Got it, I'll remove that flag.

Reply:
Updated, let me know if it's ok!